### DISCUSSION TITLE
Difference between BaseChatModel and BaseLLM for text extraction

### DISCUSSION THREAD

**Discussion by konstantinoskalyfommatos** [eyurtsev/kor#315] [General]
In theory, would one of those 2 work better for text extraction, if the model is the same? 

from langchain_community.llms.ollama import Ollama

llm = Ollama(model="llama3.1", temperature=0)
---------------------------
from langchain_ollama import ChatOllama

llm = ChatOllama(model="llama3.1", temperature=0)

**Comment by eyurtsev** [comment]
I wrote kor before chat models were a big thing, so prompts are probably better tuned for llms all else being equal. 

Otherwise I'd use langchain_ollama since that's the official integration for ollama.
